itests/manual_onboarding_test.go

```diff
@@ -115,6 +115,21 @@ func TestManualCCOnboarding(t *testing.T) {
 			// Wait till both miners' sectors have had their first post and are activated and check that this is reflected in miner power
 			waitTillActivatedAndAssertPower(ctx, t, minerB, bRespCh, bSectorNum, uint64(defaultSectorSize), withMockProofs)
 			waitTillActivatedAndAssertPower(ctx, t, minerC, cRespCh, cSectorNum, uint64(defaultSectorSize), withMockProofs)
+
+			// Miner B has activated the CC sector -> upgrade it with snapdeals
+			// Note: We can't activate a sector with mock proofs as the WdPost is successfully disputed and so no point
+			// in snapping it as snapping is only for activated sectors
+			if !withMockProofs {
+				respCh := minerB.SnapDealWithRealProofs(ctx, kit.TestSpt, bSectorNum)
```

Comment:
We have a wdPostLoop running already for minerB don't we?

Maybe when we call `minerB.OnboardCCSectorWithRealProofs(ctx, kit.TestSpt)` in the test, we give it a cancellable context and then cancel that before we start a new loop. The tricky bit would be that we want the original post loop to continue right up until we submit the `ProveReplicaUpdates3`, and the timing between doing that and starting a new wdPostLoop is embedded in `SnapDealWithRealProofs`. It could be enough to pass in a `cancelPreviousPostLoop context.CancelFunc` to `SnapDealWithRealProofs` and get it to call that just after submitting the PRU3 and prior to starting a new loop? What do you think?

Reply:
As discussed offline, we're figuring out if we need to test WdPosts for snapped sectors explicitly.